apps/web/frontend/components/bookWorkHourModal.tsx

```diff
@@ -124,14 +123,6 @@ export const BookWorkHourModal = (props: BookWorkHourModalProps): JSX.Element =>
   const [currentProjectId, currentTaskId] = watch(['projectId', 'taskId'])
   const selectedProject = data?.teamBySlug.projects.find((project) => project.id === currentProjectId)
 
-  useEffect(() => {
-    const isTaskFromSelectedProject = selectedProject?.tasks.some((task) => task.id === currentTaskId)
-    if (!isTaskFromSelectedProject) {
-      // eslint-disable-next-line unicorn/no-useless-undefined
-      setValue('taskId', '')
-    }
-  }, [currentProjectId, selectedProject])
-
```

Comment:
I think this code is necessary.

When I change the project in the select input, the task select should display "Please Select".

This is broken with the change here.